Frontend/src/index.jsx

```diff
@@ -1,35 +1,47 @@
 // External Styles (this is probably not the best way to load this?)
 import '@thewca/wca-components/dist/index.esm.css'
+import 'fomantic-ui-css/semantic.css'
+import './global.scss'
 import React from 'react'
 import { createRoot } from 'react-dom/client'
-import { createBrowserRouter, Link, RouterProvider } from 'react-router-dom'
+import { createBrowserRouter, RouterProvider } from 'react-router-dom'
 import Register from './pages/register'
 import Registrations from './pages/registrations'
+import PageHeader from './ui/Header'
+import PageFooter from './ui/Footer'
 
 const router = createBrowserRouter([
   {
     path: '/',
     element: (
-      <div>
-        <Link to="/register"> Register </Link>
-        <Link to="/registrations"> Registrations</Link>
-      </div>
+      <>
+        <PageHeader />
+        <PageFooter />
+      </>
     ),
   },
```

Comment:
Can avoid repeating the header and footer with something like
```
{
  path: '/',
  element: <><PageHeader /><Outlet /><PageFooter /></>,
  children: [
    { path: '/register', element: <main><Register /></main>},
    { path: '/registrations', element: <main><Registrations /></main> },
  ],
```

Reply:
done